### ISSUE TITLE
Assignment In Condition Inspection Not Working.

### ISSUE THREAD

**Issue by WiXSL** [kalessil/phpinspectionsea#1051]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug                          |
| Plugin         | Php Inspections (EA Extended) v3.0.6.1 |
| Language level | PHP 7.0.24

The code inside the following function doesn't trigger any issues.

```
function testAssignmentInCondition() {
    $ret = false;

    // Assignment in condition issue '$chkInt = 0'
    if (($chkInt = 0) || $chkInt == 1) {
        $ret = true;
    }

    return $ret;
}
```

--

"Assignment in condition" error should be triggered.
If you remove the parentheses from the expression '$chkInt = 0' it works as expected.

--

Environment details
---
Information from Help -> 
PhpStorm 2018.3 EAP
Build #PS-183.2940.15, built on September 26, 2018
PhpStorm EAP User
Expiration date: October 26, 2018
JRE: 1.8.0_152-release-1293-b14 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by kalessil** [comment]
Thank you for reporting @WiXSL, unfortunately, the mentioned inspection is not from us. Please report to PhpStorm team.
